### ISSUE TITLE
Create a PECL package for auto-instrumentation

### ISSUE THREAD

**Issue by bobstrecansky** [open-telemetry/opentelemetry-php#846]
Before opening a feature request against this repo, consider whether the feature should/could be implemented in the [other OpenTelemetry client libraries](https://github.com/open-telemetry). If so, please [open an issue in opentelemetry-specification](https://github.com/open-telemetry/opentelemetry-specification/issues/new) first.

We'd like to make a [PECL](https://pecl.php.net/) extension for auto-instrumentation.  We need to make sure it's in more of a production-ready state before we can do this.  This ticket is a meta-placeholder for the future work of creating this package.

**Comment by brettmc** [comment]
Found this handy guide to registering and then creating a PECL package: https://www.mediawiki.org/wiki/How_to_PECL

**Comment by brettmc** [comment]
Both Przemek and I have submitted pecl applications, and I've sent a follow-up email to the pecl mailing list.
It's a very very quiet and non-responsive list, so we might be waiting a long time for something to happen...

**Comment by bobstrecansky** [comment]
Followed up with @morrisonlevi in slack, hopefully he can point us in the correct direction.

**Comment by brettmc** [comment]
I sent a polite email to Chrisoph today (who is sometimes responsive on pecl lists), which was suggested in the guide I referenced up-thread...

**Comment by derickr** [comment]
I've opened your account.

**Comment by pdelewski** [comment]
@derickr Could you activate my account `pdelewski` as well?

**Comment by derickr** [comment]
Done.

**Comment by andypost** [comment]
https://pecl.php.net/package/opentelemetry published, thank you!
